components/cam/bld/config_files/horiz_grid.xml

```diff
@@ -26,25 +29,52 @@
 <horiz_grid dyn="fv"    hgrid="4x5"        nlat="46"   nlon="72"   />
 <horiz_grid dyn="fv"    hgrid="10x15"      nlat="19"   nlon="24"   />
 
-<horiz_grid dyn="se"    hgrid="ne2np4"     ncol=""         csne="2"    csnp="4"  />
-<horiz_grid dyn="se"    hgrid="ne4np4"     ncol="866"      csne="4"    csnp="4"  />
-<horiz_grid dyn="se"    hgrid="ne5np8"     ncol=""         csne="5"    csnp="8"  />
-<horiz_grid dyn="se"    hgrid="ne7np8"     ncol=""         csne="7"    csnp="8"  />
-<horiz_grid dyn="se"    hgrid="ne10np4"    ncol=""         csne="10"   csnp="4"  />
-<horiz_grid dyn="se"    hgrid="ne11np4"    ncol="6536"     csne="11"   csnp="4"  />
-<horiz_grid dyn="se"    hgrid="ne16np4"    ncol="13826"    csne="16"   csnp="4"  />
-<horiz_grid dyn="se"    hgrid="ne16np8"    ncol=""         csne="16"   csnp="8"  />
-<horiz_grid dyn="se"    hgrid="ne21np4"    ncol=""         csne="21"   csnp="4"  />
-<horiz_grid dyn="se"    hgrid="ne30np4"    ncol="48602"    csne="30"   csnp="4"  />
-<horiz_grid dyn="se"    hgrid="ne30np8"    ncol=""         csne="30"   csnp="8"  />
-<horiz_grid dyn="se"    hgrid="ne60np4"    ncol="194402"   csne="60"   csnp="4"  />
-<horiz_grid dyn="se"    hgrid="ne120np4"   ncol="777602"   csne="120"  csnp="4"  />
-<horiz_grid dyn="se"    hgrid="ne240np4"   ncol="3110402"  csne="240"  csnp="4"  />
-<horiz_grid dyn="se"    hgrid="ne0np4_arm_x8v3_lowcon"   ncol="92558"   csne="0"    csnp="4"  />
-<horiz_grid dyn="se"    hgrid="ne0np4_conus_x4v1_lowcon"   ncol="89147"   csne="0"    csnp="4"  />
-<horiz_grid dyn="se"    hgrid="ne0np4_svalbard_x8v1_lowcon"   ncol="71912"   csne="0"    csnp="4"  />
-<horiz_grid dyn="se"    hgrid="ne0np4_sooberingoa_x4x8v1_lowcon"   ncol="105707"   csne="0"    csnp="4"  />
-<horiz_grid dyn="se"    hgrid="ne0np4_enax4v1"          ncol="78788"    csne="0"    csnp="4"  />
-<horiz_grid dyn="se"    hgrid="ne0np4_twpx4v1" ncol="81434" csne="0" csnp="4"  />
+<!-- Spectral Element Dynamics -->
+<!-- ncol = # of columns in the physics grid -->
+<!-- if npg=0 then physics is done on GLL nodes -->
+<!-- if npg=n>0 then physics is done FV grid with nxn cells per element -->
```

Comment:
I'm just going to note here that offline you mentioned putting the formula for calculating the ncol for each grid (for future reference if new grids are introduced).

Reply:
yes, this was added in a recent commit.